drivers/XE_SR_ERRORCODES.xml

```diff
@@ -881,6 +881,12 @@
             <value>460</value>
         </code>
 
+        <code>
+            <name>WipeFS</name>
+            <description>Failed to wipe pre-existing filesystem signature.</description>
```

Comment:
A lot of descriptions don't have a period at the end. Maybe we need to add a contribution in the upstream repository to standardize this situation.